docs/topics/i18n/translation.txt

```diff
@@ -70,6 +70,20 @@ as a shorter alias, ``_``, to save typing.
        global ``_()`` function causes interference. Explicitly importing
        ``ugettext()`` as ``_()`` avoids this problem.
 
+    .. admonition:: What functions may be aliased as ``_``?
+
+        Only these functions can be imported as ``_``:
```

Comment:
Perhaps: "Because of how xgettext works, only functions that take a single string argument can be ..."